### ISSUE TITLE
Severely distorted video in Unity/Carboard/iOS 360 video player (dpi issue?)

### ISSUE THREAD

**Issue by 1Presidio** [googlevr/gvr-unity-sdk#827]
I made an application that can view 360 photos or videos. It is build on unity and works fine on Android devices. However when I try to do the same project in iOS there are servious

Found using:
 * GVR SDK for Unity v1.110.0
 * Unity version: 2017.3.0f3 Personal
 * iPhone 5s and iPhone X iOS 11.2.1

Steps to reproduce the issue:
 1. Build the app on Unity with VR enabled and the Cardboard SDK checked
 2. Added a GvrEditorEmulator Unity Package from the prefab folder
 3. After doing *Build and Run* it builds on Unity and opens the xCode project and runs
 4. Finish codesigning and the app is installed
 5. The resulting image is distorted and misaligned. The windows for each eye are contorted and the screens are misaligned with one looking "backwards or sideways" and one looking straight. They should be stereoscopic with similar images.

This is really frustrating and I have had no luck getting help from other posts. I have a feeling it is a DPI issue where it things the dimensions are wrong and does not render it correctly. Please let me know if you need more details.
![img_1618](https://user-images.githubusercontent.com/21327708/34589685-e24f3f3e-f166-11e7-8604-08da22e02730.PNG)

![img_1617](https://user-images.githubusercontent.com/21327708/34589615-86c55aae-f166-11e7-9001-aec72d7e8cd1.PNG)

**Comment by clod76** [comment]
i have the same issue on iphone 6 !

**Comment by fredsa** [comment]
Looks like the portrait width was used to determine the size of the stereo view, but that the video is being stretched horizontally. Similar to what's causing issue #602.

Can you try the [workaround](https://github.com/googlevr/gvr-unity-sdk/issues/602#issuecomment-339157524) I listed there?

**Comment by 1Presidio** [comment]
@clod76 @fredsa 

After some careful trial and error debugging (because figuring that since my app works for android but not iOS it must be an xcode issue) I realized there is some sort of bug with the device orientation when all the boxes are checked. Thus when building under the Deployment Info Tab under Device Orientation only select one of them. Then I got it to work.

I believe the error is caused by the app whether GoogleSDK or Unity thinking that the phone is oriented multiple ways or something and thus incorrectly renders the screen.
